include/zephyr/arch/cache.h

```diff
@@ -349,6 +349,9 @@ void *arch_cache_uncached_ptr_get(void __sparse_cache *ptr);
 #define cache_uncached_ptr(ptr) arch_cache_uncached_ptr_get(ptr)
 #endif /* CONFIG_CACHE_DOUBLEMAP */
 
+
+int arch_cache_init(void);
```

Comment:
As we don't use the return value - let's drop it:

```suggestion
void arch_cache_init(void);
```

Reply:
fixed